toxics/toxic_test.go

```diff
@@ -99,30 +99,22 @@ func ToxicToJson(t *testing.T, name, typeName, stream string, toxic toxics.Toxic
 		return nil
 	}
 
-	marshal := make(map[string]interface{})
-	err = json.Unmarshal(data, &marshal)
-	if err != nil {
-		if t != nil {
-			t.Errorf("Failed to unmarshal toxic (2): %v", toxic)
-		}
-		return nil
-	}
-	marshal["type"] = typeName
-	if name != "" {
-		marshal["name"] = name
-	}
-	if stream != "" {
-		marshal["stream"] = stream
+	data := map[string]string{
```

Comment:
Actually, this would be another option for `MarshalJSON()` if we do keep it around, would be much cleaner at least.